invenio_files_rest/helpers.py

```diff
@@ -85,7 +102,14 @@ def send_stream(stream, filename, size, mtime, mimetype=None, restricted=True,
 
 
 def make_path(base_uri, path, filename, path_dimensions, split_length):
-    """Generate a path as base location for file instance."""
+    """Generate a path as base location for file instance.
+
+    :param base_uri: The base URI.
+    :param path: The relative path.
+    :param path_dimensions: How many peaces the path should be splitted.
```

Comment:
Number of pieces the path should be split into. (although I'm not sure the word "pieces" is well chosen... maybe chunks/sub-paths? To correct also line 110)